os-depends

```diff
@@ -33,7 +33,7 @@ gnome-shell (>= 3.22.3)
 gnupg
 # Could just be i386 amd64, but theoretically supports other arches
 grub2 [!armhf]
-grub-efi-amd64-image-signed [amd64] | grub-efi-amd64-image [i386]
+grub-efi-amd64-image-signed [amd64] | grub-efi-ia32-image [i386]
```

Comment:
I don't think this is right: we want to have both `grub-efi-amd64-image-signed` (for the EFI-64 bootloader) and `grub-efi-ia32-image` (for the EFI-32 bootloader) in the image, no?

Reply:
ok, I'm a bit confused then. My reasoning here is that in the `amd64` image (i.e.) we only need `grubx64.efi`. For the `i386` images only `bootia32.efi`. Why do we want to have both in general?

Reply:
I'm not fully educated in EFI, but I think @carlocaione is right. On a 64 bit system (amd64), you would only ever need `grubx64.efi`, right? Whereas on a 32 bit system (i386), you would only need `bootia32.efi`. But, of course, these programs don't run on the OS, so maybe I'm wrong.

That said, I think `bootia32.efi` was moved to a separate package entirely so that it wasn't shipped on amd64. If `grub-efi-ia32-image` is installed on amd64, then you'd still have it, which would defeat the purpose.

Reply:
I think the argument you could make is that you might want both 32 and 64 bit EFI loaders on i386 because you might be running a 32 bit OS on a 64 bit system. However, you couldn't say the opposite - 64 bit OS on a 32 bit system. So, I think the most pedantically correct would likely be:
```
grub-efi-amd64-image-signed [amd64 i386]
grub-efi-ia32-image [i386]
```

Reply:
this makes sense. I'll respin the PR.

Reply:
No, we explicitly want the 32-bit EFI loader in the 64-bit OS image. This is because there are machines that have a 64-bit CPU but 32-bit EFI firmware.

Reply:
https://phabricator.endlessm.com/T13970

Reply:
jesus, why such a monster does even exist. Alright, then, it looks like we must ship both the packages on both the architectures.